app/models/concerns/rad_user.rb

```diff
@@ -365,7 +371,12 @@ def notify_user_approved
     end
 
     def notify_user_approved_user
-      RadMailer.your_account_approved(self).deliver_later
+      raise 'missing approved_by' if approved_by.blank?
```

Comment:
I think this is only applicable to sign up apps like ABN, which had overridden behavior that also sent a text message to new users when they are approved. Made it standard here which doesn't seem like it will hurt anything.